src/devtools/views/Components/OwnersStack.css

```diff
@@ -38,7 +36,48 @@
   outline: none;
 }
 
+.ElementsBar {
+  flex: 1 0 auto;
+}
+
+.ElementsBarSelectedOnly {
+  margin-left: 0.25rem;
+}
+
+.ElementsBarSelectedOnly .Component {
+  visibility: hidden;
```

Comment:
We are hiding all of the components that are not the active ones for two main reasons:
- to still have the ElementsBar stretched
- calculate the width correctly when we are resizing